network-api/networkapi/settings.py

```diff
@@ -114,6 +119,12 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = root()
 
+# Basket client configuration
+BASKET_URL = env('BASKET_URL')
+BASKET_API_KEY = env('BASKET_API_KEY')
```

Comment:
```suggestion
```
I know we agreed to add this originally, but I don't think we'll ever need this in donate, so it's probably good to remove.